app/src/main/java/com/osunick/voicerecorder/ui/compose/VoiceLogsScreen.kt

```diff
@@ -211,13 +213,27 @@ fun VoiceLogList(
             items(count = lazyPagerItems.itemCount) { index ->
                 lazyPagerItems[index]?.let { message ->
 
-                    Column(modifier = Modifier.padding(4.dp)) {
-                        Text(
-                            modifier = Modifier,
-                            text = formatDateTime(message.dateTime),
-                            color = onSurfaceVariantLight,
-                            style = Typography.labelMedium
-                        )
+                    Row(modifier = Modifier.padding(4.dp)) {
+                        Column() {
+                            Text(
+                                modifier = Modifier
+                                    .padding(horizontal = 8.dp, vertical = 0.dp)
```

Comment:
Nit: I think you can just leave out vertical when it's zero, since that's the default